struct/GiveawayManager.ts

```diff
@@ -0,0 +1,313 @@
+import Giveaway from "./Giveaway";
+import moment from "moment";
+import {
+  Client,
+  Message,
+  MessageEmbed,
+  MessageReaction,
+  TextChannel,
+  User,
+} from "discord.js";
+import { stripIndent } from "common-tags";
+import Enmap from "enmap";
+import _ from "lodash";
+import { DefaultOptions } from "./interfaces";
+const giveaways = new Enmap({
+  name: "giveaways",
+  autoFetch: true,
+  fetchAll: true,
+});
+const messages = new Enmap({
+  name: "messages",
+  fetchAll: true,
+  autoFetch: true,
+});
+
+export { giveaways };
```

Comment:
```suggestion
export { giveaways, messages };
```
Just for later, incase people want to make a command to check how many messages they have